### PR TITLE
Fixed command explanation and small formatting changes around cron logging.

### PR DESCRIPTION
**Pull request by dobooth**
## This PR is a:

- [ ] Content update

## Summary

When this pull request is merged, it will...

Clarify instructions for running syslog commands.

### CODE REVIEW

File: guides/v2.3/config-guide/cli/logging.md [hunk 1]
```diff
@@ -13,83 +13,84 @@ functional_areas:
...
 
-#### To enable syslog logging:
+### To enable syslog logging
 
-1. Use the `config:set` command to change the `dev/syslog/syslog_logging` database value to `1`.
+1. Use the `config:set` command to enable syslog logging.
 
     ```bash
-    bin/magento config:set dev/syslog/syslog_logging 1
+       bin/magento config:set dev/syslog/syslog_logging 1
```
Comment (jeff-matthews) [inline guides/v2.3/config-guide/cli/logging.md]: Isn't this the correct command syntax?

```bin/magento setup:config:set --enable-syslog-logging=true```

File: guides/v2.3/config-guide/cli/logging.md [hunk 2]
```diff
@@ -13,83 +13,84 @@ functional_areas:
...
 
-#### To disable syslog logging:
+### To disable syslog logging
 
-1. Use the `config:set` command to change the `dev/syslog/syslog_logging` database value to `0`.
+1. Use the `config:set` command to disable syslog logging.
 
     ```bash
-    bin/magento config:set dev/syslog/syslog_logging 0
+       bin/magento config:set dev/syslog/syslog_logging 0
```
Comment (jeff-matthews) [inline guides/v2.3/config-guide/cli/logging.md]: Isn't this the correct command syntax?

```bin/magento setup:config:set --enable-syslog-logging=false```

**Comment by keharper** [comment]
I think the command changed for 2.3.1.

**Comment by jeff-matthews** [comment]
Also, if this is a 2.3.1 change, we should merge with the `2.3.1-integration` branch; not `master`.

File: guides/v2.3/config-guide/cli/logging.md (hunk 1 above)
Comment (dobooth) [reply guides/v2.3/config-guide/cli/logging.md]: Thanks.

File: guides/v2.3/config-guide/cli/logging.md (hunk 2 above)
Comment (dobooth) [reply guides/v2.3/config-guide/cli/logging.md]: Thanks.

**Comment by jeff-matthews** [comment]
This is weird... The commands are correct, but the changes make it look like you never touched them. It must've had something to do with your force push.

**Comment by jeff-matthews** [comment]
This [commit](https://github.com/magento/devdocs/commit/e4845499cfcaed513e1bab88540caebae534c482#diff-b95298efe2674847d32b3dbad3f9429c) added the correct command syntax to the `2.3.1-integration` branch.

**Pull request closed without merging**